src/pages/documentation/platform-assurance/tutorials/optimize-configuration-view.md

```diff
@@ -0,0 +1,35 @@
+# Optimize Setup Validation view
+
+## Overview
+
+You may use the Optimize Setup Validation view in order to quickly see if your app has been successfully set up to use AEP Optimize. The view validates all pre-requisites have been met, and offers feedback on how to fix your setup in cases where requirements have been missed.
+
+## Getting Started
+
+To use this view, complete the following steps:
+
+1. [Set up Adobe Experience Platform Assurance](../set-up.md)
+2. [Create](./index.md#creating-sessions) and [connect](./index.md#connecting-to-a-session) to an Assurance session
+3. In the **Home** view, select **Validate Setup**
+
+![Validate Setup](./assets/optimize-configuration/optimize-setup-view.png)
+
+## Datastream Setup Validation
+
+At the top of the Optimize Setup Validation view, Platform Assurance inspects your current app's datastream, and looks to see if Target or Offer Decisioning has been enabled. In order to use optimize, at least one of extensions needs to be enabled inside your datastream.
+
+![Validate Datastream Section](./assets/optimize-configuration/datastream-validation.png)
+
+To enable Target in your datastream, go to the datastream view for your app inside the Data Collection UI, and ensure that Adobe Target has been added and enabled.
+
+![Enable Target Datastream](./assets/optimize-configuration/enable-target-datastream.png)
+
+To enable Offer Decisioning in your datastream, go to the datastream view for your app inside the Data Collection UI, and then select Edit inside the actions menu for Adobe Experience Platform. Inside, you will see a checkbox that shows whether Offer Decisioning has been enabled.
```

Comment:
Suggest rewording this section: "To enable Offer Decisioning in your datastream, go to datastream view for your application in the Data Collection UI, and select Edit in the actions menu for Adobe Experience Platform. The toggle shows if Offer Decisioning is enabled."